intake_stac/catalog.py

```diff
@@ -331,15 +331,25 @@ def __init__(self, key, item, stacked=False):
     def _get_driver(self, entry):
         drivers = {
             'application/netcdf': 'netcdf',
+            'application/x-netcdf': 'netcdf',
+            'application/parquet': 'parquet',
+            'application/x-parquet': 'parquet',
+            # 'application/x-hdf5': '',
+            # 'application/x-hdf': '',
```

Comment:
There is not a specific driver for hdf at this point so we could either leave this off or try the netcdf driver, which will work on a subset of hdf5 datasets.

Reply:
Lets try maybe it will work for some.